presto-main/src/test/java/io/prestosql/SessionTestUtils.java

```diff
@@ -15,16 +15,21 @@
 
 import io.prestosql.client.ClientCapabilities;
 
+import java.time.Instant;
+
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static io.prestosql.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
 import static io.prestosql.testing.TestingSession.testSessionBuilder;
 import static java.util.Arrays.stream;
 
 public final class SessionTestUtils
 {
+    public static final Instant SESSION_START_TIME = Instant.parse("2007-12-03T10:15:30.00Z");
+
     public static final Session TEST_SESSION = testSessionBuilder()
             .setCatalog("tpch")
             .setSchema(TINY_SCHEMA_NAME)
+            .setStartTime(SESSION_START_TIME.toEpochMilli())
```

Comment:
We either add this to `io.prestosql.testing.TestingSession#testSessionBuilder()` for all tests or add in specific tests only. 

Also, i don't know why `io.prestosql.SessionTestUtils#TEST_SESSION` is providing different defaults than `io.prestosql.testing.TestingSession#testSessionBuilder()`. Maybe we can remove `SessionTestUtils`?

Reply:
done,

> Maybe we can remove `SessionTestUtils`?

This is probably a good idea, but not part of this PR